client/css/style.css

```diff
@@ -818,6 +819,11 @@ button,
 	color: #2ecc40;
 }
 
+#chat .ctcp .from:before {
+	font-family: FontAwesome;
+	content: "\f0f6";
+}
+
```

Comment:
Would you mind keeping the CTCP commit in a different PR?
Not at all that I'd be against it, but the PR is already very long with about 1k changes and 40 commits and since I'd need to read about it (I don't know anything about it yet), I think this should go separate to expedite the whole thing more quickly.